openrtb_ext/imp_appnexus_Keywords_test.go

```diff
@@ -0,0 +1,35 @@
+package openrtb_ext
+
+import (
+	"encoding/json"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestKeywordsUtilUnmarshalJSON(t *testing.T) {
+	type keywords struct {
+		Keywords Keywords `json:"keywords"`
+	}
+
+	t.Run("dynamic-json", func(t *testing.T) {
```

Comment:
Please consider re-writing these as table driven tests as discussed offline? Also, consider adding test cases for error conditions such as empty JSON and invalid JSON

Reply:
Done in [034e995](https://github.com/prebid/prebid-server/pull/2649/commits/034e9950d26f5223e94b5341f8333f77cb32dbe6)